bin/node/runtime/src/lib.rs

```diff
@@ -1972,7 +1972,27 @@ impl_runtime_apis! {
 
 	impl pallet_nomination_pools_runtime_api::NominationPoolsApi<Block, AccountId, Balance> for Runtime {
 		fn pending_rewards(member_account: AccountId) -> Balance {
-			NominationPools::pending_rewards(member_account).unwrap_or_default()
+			NominationPools::api_pending_rewards(member_account).unwrap_or_default()
+		}
+
+		fn points_to_balance(pool_id: u32) -> Balance {
+			NominationPools::api_points_to_balance(pool_id)
+		}
+
+		fn balance_to_points(pool_id: u32, new_funds: Balance) -> Balance {
+			NominationPools::api_balance_to_points(pool_id, new_funds)
+		}
+	}
+
+	impl pallet_staking_runtime_api::StakingApi<Block, Balance> for Runtime {
+		fn inflation_rate() -> Perquintill {
```

Comment:
Inflation rate API added here. We use the same `ideal_stake` and `falloff` values that are plugged in to the reward curve.